keymaster/ta/mbedtls_proxy.c

```diff
@@ -736,8 +728,7 @@ static TEE_Result mbedTLS_attest_key_cert(mbedtls_pk_context *issuer_key,
 	}
 
 	ret = mbedtls_x509write_crt_set_key_usage(&crt,
-					    MBEDTLS_X509_KU_DIGITAL_SIGNATURE |
-					    MBEDTLS_X509_KU_KEY_CERT_SIGN);
+					    key_usage);
```

Comment:
This change also requires generating proper basic constraints with CA set to false in line 707: 
[https://tools.ietf.org/html/rfc5280#section-4.2.1.9](rfc5280)